hazelcast-client-legacy/src/test/java/com/hazelcast/client/replicatedmap/ClientReplicatedMapListenerTest.java

```diff
@@ -115,6 +117,50 @@ public void run() throws Exception {
         });
     }
 
+
+    @Test
+    public void testListenToKeyForEntryAdded() throws Exception {
+        HazelcastInstance instance = factory.newHazelcastInstance();
+        HazelcastInstance client = factory.newHazelcastClient();
+        String mapName = randomMapName();
+        ReplicatedMap<Object, Object> replicatedMap = client.getReplicatedMap(mapName);
+        final EventCountingListener listener = new EventCountingListener();
+        replicatedMap.addEntryListener(listener, 1);
+        replicatedMap.put(1, 1);
+        replicatedMap.put(2, 2);
+        assertTrueEventually(new AssertTask() {
```

Comment:
Testing flavor; I add line of white to quickly find the 'do' and the verification phase.